src/client/datascience/interactive-window/nativeInteractiveWindow.ts

```diff
@@ -458,18 +454,7 @@ export class NativeInteractiveWindow implements IInteractiveWindowLoadable {
                     if (cell.cell_type === 'code') {
                         // Then send the combined output to the UI
                         const converted = cell.outputs.map(cellOutputToVSCCellOutput);
-                        void temporaryExecution.replaceOutput(converted).then(() => {
```

Comment:
I believe https://github.com/microsoft/vscode/pull/129014 makes this code redundant, @rebornix